Make `allowUrls` and `denyUrls` more precise (#10472)
getsentry/sentry-docs: includes/platforms/configuration/options/deny-urls.mdx

> An array of strings or regex patterns that match the URLs of scripts where errors have been created.
> Only errors that have been created on these URLs will not be sent sent to Sentry.
> If you use this option, Errors whose top stack frame file URL contains (string) or matches (regex) at least one entry in the array will not be sent.
> 
> This matching logic applies for captured exceptions, not raw message events. By default, all errors are sent.

Reviewer: ```suggestion
An array of strings or regex patterns that match the URLs of scripts where errors have been created. Errors that have been created on these URLs won't be sent to Sentry.
If you use this option, at least one entry in the array in which an error's top stack frame file URL contains (string) or matches (regex) won't be sent.

This matching logic applies to captured exceptions not raw message events. By default, all errors are sent.
```

Author: Also took another stab here: b711447a0d9e72b20df090acf99cefbc063b5465
